tests/classes/tsrequest_test.php

```diff
@@ -61,6 +61,17 @@ public function test_connection_test() {
             ->setConstructorArgs([TURNITINSIM_ENDPOINT_WEBHOOKS])
             ->getMock();
 
+        //Test connection should return failed if url is missing
```

Comment:
This might sound nitpicky but Moodle's code prechecker requires comments end with a fullstop. It won't break anything, just shows up as a warning on the check that is listed on Moodle.org